webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -118,29 +118,30 @@ export default class CreateIssueModal extends PureComponent {
     handleIssueDescriptionChange = (issueDescription) => this.setState({issueDescription});
 
     renderIssueAttributeSelectors = () => {
-        if (!this.state.repo || (this.state.repo.permissions && !this.state.repo.permissions.push)) {
-            return null;
-        }
+        const otherOptionsDisabled = !this.state.repo || (this.state.repo.permissions && !this.state.repo.permissions.push);
 
         return (
             <>
                 <GithubLabelSelector
-                    repoName={this.state.repo.name}
+                    repoName={this.state.repo ? this.state.repo.name : ''}
```

Comment:
Can we use a variable above to remove some of the code duplication among the different components here?

```js
const repoName = this.state.repo ? this.state.repo.name : '';
```

```suggestion
                    repoName={repoName}
```

Reply:
done